src/levels/rampup/interactiveRebase.js

```diff
@@ -10,6 +10,7 @@ exports.level = {
     "en_US": "you can use either branches or relative refs (HEAD~) to specify the rebase target",
     "es_AR": "podés usar tanto ramas como referencias relativas (HEAD~) para especificar el objetivo del rebase",
     "es_ES": "puedes usar tanto ramas como referencias relativas (HEAD~) para especificar el objetivo del rebase",
+    "es_MX": "puedes usar tanto ramas como referencias relativas (HEAD~) para especificar el objetivo del rebase",
```

Comment:
@pcottle does the translation system support language fallbacks? For example, ex_MX could have es_ES as a fallback, pt_BR could have a pt_PT fallback (or vice-versa) and in cases like this where the strings are identical, one could leave the string missing and let the system pick up the fallback string.

There are several advantages to such a system:
- We would avoid duplicating the same content in multiple translations
- It would be less likely that related translations would go out of sync
- Changes to the fallback language would be automatically picked up by the dependent languages
- The effort to add a new translation for language variants would be reduced
- Users would see missing strings as a language they're likely to understand better, rather than English

Reply:
For the record, I would gladly contribute a pt_PT translation if I could lean on the existing pt_BR translation and only add the strings that would actually be different.